app/command_loadtest.go

```diff
@@ -28,25 +29,31 @@ var usage = `Mattermost testing commands to help configure the system
 
 	Users - Add a specified number of random users with fuzz text to current team.
 		/test users [fuzz] <Min Users> <Max Users>
-		
+
 		Example:
 			/test users fuzz 5 10
 
 	Channels - Add a specified number of random channels with fuzz text to current team.
 		/test channels [fuzz] <Min Channels> <Max Channels>
-		
+
 		Example:
 			/test channels fuzz 5 10
 
 	Posts - Add some random posts with fuzz text to current channel.
 		/test posts [fuzz] <Min Posts> <Max Posts> <Max Images>
-		
+
 		Example:
 			/test posts fuzz 5 10 3
 
+	Post - Add post to a channel as another user.
+		/test post u=@username p=passwd c=~channelname t=teamname "message"
```

Comment:
Is there some convention that you're following with the `u=` prefixes on the arguments? They look a bit weird to me, but if we don't have any other convention to follow, they're probably fine as is

Reply:
None, it's my random thought to use the first letter (e.g. "u" for "user").  I can't think of a better way, open for suggestion.

Reply:
This is fine then. We can always change this if we come up with some better convention